@xen-orchestra/lite/src/views/host/HostNetworkView.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <PageUnderConstruction />
+</template>
+
+<script lang="ts" setup>
+import PageUnderConstruction from '@/components/PageUnderConstruction.vue'
+import { usePageTitleStore } from '@/stores/page-title.store'
+import { useI18n } from 'vue-i18n'
+
+usePageTitleStore().setTitle(useI18n().t('dashboard'))
```

Comment:
You can set the real page name, so we won't forget to update it when we implement this page. Remember to add the translations in the locales’ files:
```suggestion
usePageTitleStore().setTitle(useI18n().t('network'))
```